docs/conf.py

```diff
@@ -31,6 +32,16 @@
     "discussion": ("https://github.com/zarr-developers/virtualizarr/discussions/%s", "D%s"),
 }
 
+# Example configuration for intersphinx: refer to the Python standard library.
+# use in refs e.g:
+# :ref:`comparison manual <python:comparisons>`
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+    "numpy": ("https://numpy.org/doc/stable/", None),
+    "zarr": ("https://zarr.readthedocs.io/en/stable/", None),
+    "xarray": ("https://docs.xarray.dev/en/stable/", None),
```

Comment:
Add obstore too? Or you want to add it in the other PR?

Reply:
Yeah I figured I would add it in the other PR, since it's not used anywhere in VirtualiZarr yet.